addons/crm/views/crm_lead_views.xml

```diff
@@ -125,8 +125,20 @@
                                     <field name="country_id" placeholder="Country" class="o_address_country" options='{"no_open": True, "no_create": True}'/>
                                 </div>
                                 <field name="website" widget="url" placeholder="e.g. https://www.odoo.com"/>
-                                <field name="lang_id"/>
+                                <field name="lang_active_count" invisible="1"/>
                                 <field name="lang_code" invisible="1"/>
+                                <label for="lang_id" attrs="{'invisible': [('lang_active_count', '&lt;=', 1)]}"/>
+                                <div class="o_row" attrs="{'invisible': [('lang_active_count', '&lt;=', 1)]}">
+                                    <field name="lang_id" options="{'no_quick_create':True,'no_create_edit':True,'no_open': True,}"/>
```

Comment:
Linting: don't forget spaces after "," and ":" (``options="{'no_quick_create': True, 'no_create_edit': True, 'no_open': True}"``